CHANGELOG.md

```diff
@@ -51,8 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1313](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1313))
 - Fix uninstrumentation of existing app instances in FastAPI
   ([#1258](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1258))
-- Fix uninstrumentation of existing app instances in falcon
-  ([#1341]https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1341)
```

Comment:
Why remove this? And also move your fix to unreleased section.

Reply:
That's an error on my side